.buildkite/inference-benchmark-pipeline.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -euf
+
+# This script is executed by Buildkite to generate pipeline YAML string.
+# Document: https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines
+#
+# Usage: CONVERT_RESULT_PATH=gs://path/to/result.tgz ./inference-benchmark-pipeline.sh
+
+# test cases to run inference
+# "<buildkite agent-type> <model filename>"
+TEST_CASES=(
+    "de10nano       lib/lib_arm.so"
+    "de10nano       lib/lib_fpga.so"
+
+    "jetson-nano    minimal_graph_with_shape.pb"
+    "jetson-tx2     minimal_graph_with_shape.pb"
+    "jetson-xavier  minimal_graph_with_shape.pb"
+
+    "raspberry-pi   lib/lib_aarch64.so"
+)
+
+# Generate setup step
+cat <<EOS
+steps:
+  - label: "inference: setup"
+    key: "setup"
+    timeout_in_minutes: "30"
+    agents:
+      - "env=benchmark"
+      - "agent-type=gcloudsdk"
+
+    artifact_paths:
+      - "convert-result.tgz"
+
+    command: |
+      gsutil -m cp "${CONVERT_RESULT_PATH}" ./
```

Comment:
I think we should add `wait` after this step to wait for this setup before running inference steps.
https://buildkite.com/docs/pipelines/wait-step

Reply:
I set `depends_on: "setup"` to inference steps.
https://buildkite.com/docs/pipelines/dependencies

Reply:
Oh! So nice! 👍